src/components/ProgressPopup/styles.module.scss

```diff
@@ -10,27 +10,34 @@
   box-shadow: 0 0 6px #aaa;
   border-radius: 4px;
 
-  .levels{

```

Comment:
Empty rule set is correctly removed, but it's still referenced in code and should be removed.